### PR TITLE
[71_10] Qt: translate button label for inputs_list_widget

### PR DESCRIPTION
**Pull request by darcy-shen**

### CODE REVIEW

File: TeXmacs/progs/generic/document-edit.scm [hunk 1]
```diff
@@ -74,7 +74,8 @@
 (tm-define (init-interactive-env var)
   (:interactive #t)
   (interactive (lambda (s) (set-init-env var s))
-    (list (or (logic-ref env-var-description% var) var) "string"
+    (list (or (logic-ref env-var-description% var) var)
```
Comment (jingkaimori) [inline TeXmacs/progs/generic/document-edit.scm:77]: Why there are two change of two blank character? These change should be moved to separate pr.

Comment (darcy-shen) [reply TeXmacs/progs/generic/document-edit.scm:77]: it is the related code path

Comment (darcy-shen) [reply TeXmacs/progs/generic/document-edit.scm:77]: it illustrates how the developer find the bug, and it happened that there are better code formatting in the code path
